### PR TITLE
Fix overrelease of dismiss completion block in UIViewController.mm

### PR DESCRIPTION
**Pull request by ehren**
Now get a decent stack for a crash with my app pointing to this (and looking at SmartTypes.h idretainp has the same releasing semantics as idretaintype)

### CODE REVIEW

**Comment by Ramu-msft** [comment]
@davelamb is added to the review. #Closed

**Comment by Ramu-msft** [comment]
@bdrlamb is added to the review. #Closed

**Comment by DHowett-MSFT** [comment]
:+1:

File: Frameworks/UIKit/UIViewController.mm [hunk 1]
```diff
@@ -1116,7 +1116,6 @@ - (void)_dismissTransitionStopped:(id)anim finished:(BOOL)finished {
 
     if (priv->_dismissCompletionBlock) {
         priv->_dismissCompletionBlock();
-        [priv->_dismissCompletionBlock release];
```
Comment (DHowett-MSFT) [inline Frameworks/UIKit/UIViewController.mm]: Good catch!

**Comment by jaredhms** [comment]
:+1:

**Comment by Ramu-msft** [comment]
:+1:

**Pull request merged**